packages/victory-selection-container/src/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { VictoryContainerProps } from "victory-core";
+
+interface VictorySelectionContainerProps extends VictoryContainerProps {
+  activateSelectedData?: boolean;
+  allowSelection?: boolean;
+  disable?: boolean;
+  onSelection?: Function;
```

Comment:
Since we know the types for the args here, I think it's possible to go more granular, e.g.:

```
interface PointsInterface = {
childName?: string | string[];
eventKey?: string | number; 
data?: any;
}

{(points: PointsInterface[], bounds: {x: number | Date, y: number | Date}[], props: any) => void}
```

(I don't know that these are the full set of types for these args, but the gist of them: https://formidable.com/open-source/victory/docs/victory-selection-container#onselection)